frontend/locales/en-US.json

```diff
@@ -345,6 +345,18 @@
         "description": "Determines which user library should be displayed on connected devices. This can be overridden for each device using profiles.",
         "selector": "Select default user"
       },
+      "EnablePlayToTracing": {
+        "text": "Enable PlayTo Tracing",
+        "description": "Enables debug logging for DLNA PlayTo"
+      },
+      "BlastAliveMessageIntervalSeconds": {
```

Comment:
Was wondering why we seemingly have two properties describing the same setting. Turns out the server team planned to deprecate and remove this setting a couple of years ago, however, never pulled through. https://github.com/jellyfin/jellyfin/blob/master/Emby.Dlna/Configuration/DlnaOptions.cs#L59

What puzzles me is the fact that setting the "correct" value with `AliveMessageIntervalSeconds` is not processed by the server.

I left both in, however, marked the working setting as deprecated as it seems to be looking at the backend code.

Reply:
This is basically to keep backwards compat with old clients (like Web).

The fact that AliveMessageIntervalSeconds is not processed is something that @jellyfin/server needs to chime in.

Reply:
I don't think anyone has worked on the DLNA code for quite a while... so who knows what will happen.

Reply:
@LPkkjHD What about opening a PR in server to attach the logic of ``BlastAliveMessageIntervalSeconds`` to ``AliveMessageIntervalSeconds``, so both do the same while the deprecation process is still going?

Reply:
This behavior is already implemented. I worked my way around it in https://github.com/jellyfin/jellyfin-vue/pull/1759/commits/e32296183dd96d7923ec80f745963ff4c0a54a50